### PR TITLE
RavenDB-8737 RQL auto complete snippets

### PR DESCRIPTION
**Pull request by fitzchak**

### CODE REVIEW

File: src/Raven.Studio/gulpfile.js [hunk 1]
```diff
@@ -234,7 +234,7 @@ gulp.task('z_mochaTests', function () {
 });
 
 gulp.task('test', [ 'z_compile:test' ], function (cb) {
-    return runSequence('z_generate-test-list', 'z_mochaTests', cb);
+    return runSequence('z_generate-test-list', /*'z_mochaTests', */cb);
```
Comment (ml054) [inline src/Raven.Studio/gulpfile.js:237]: remove or put TODO ?

**Review by ml054 — APPROVED** [review]

File: src/Raven.Studio/gulpfile.js (hunk 1 above)
Comment (fitzchak) [reply src/Raven.Studio/gulpfile.js:237]: For now I do not want to run the tests as part of the watch, but the test method should run them probably. So I commented this until later.

Comment (ml054) [reply src/Raven.Studio/gulpfile.js:237]: it is 'test' task, not 'watch' so maybe watch task is broken. Watch indeed should only compile files but not running tests. Could you fix that then? When running 'test' task we want to run tests. Current change breaks this feature.

Comment (fitzchak) [reply src/Raven.Studio/gulpfile.js:237]: Reverted, https://github.com/ravendb/ravendb/pull/3785

**Pull request merged**